File: ParteJava/ProyectoPro3/ProyectoPro3Modelo/src/main/java/pe/edu/pucp/proyectopro3/modelo/auth/Administrador.java
package pe.edu.pucp.proyectopro3.modelo.auth;

import pe.edu.pucp.proyectopro3.modelo.crm.TipoDocumento;

public class Administrador extends Usuario{
    public Administrador(int idUsuario, String nombres, String apellidos, TipoDocumento tipoDocumento, String numeroDocumento, String numeroContacto, String correo, String contrasena) {
        super(idUsuario, nombres, apellidos, tipoDocumento, numeroDocumento, correo, contrasena, numeroContacto);
    }

}

File: ParteC#/ProyectoProgramacion3/ProyectoProgramacion3Model/Model/auth/Administrador.cs
using ProyectoProgramacion3Model.Model.crm;
namespace ProyectoProgramacion3Model.Model.auth;

public class Administrador : Usuario
{
    public Administrador()
    {
    }

    public Administrador(int idUsuario, string nombres, string apellidos, TipoDocumento tipoDocumento, 
        string numeroDocumento, string numeroContacto, string correo, string contrasena) :
        base(idUsuario, nombres, apellidos, tipoDocumento, numeroDocumento, numeroContacto, correo, contrasena)
    {
    }

    public void buscarUsuario(int id)
    {
    }

    public void crearUsuario(Usuario u)
    {
    }

    public void editarUsuario(int id)
    {
    }

    public void eliminarUsuario(int id)
    {
    }
}
